app/src/main/java/mod/elfilibustero/sketch/lib/valid/VariableModifierValidator.java

```diff
@@ -0,0 +1,32 @@
+package mod.elfilibustero.sketch.lib.valid;
+
+import a.a.a.MB;
+import android.content.Context;
+import com.google.android.material.textfield.TextInputLayout;
+import java.util.regex.Pattern;
+
+public class VariableModifierValidator extends MB {
+    public static final Pattern PATTERN_MODIFIER = Pattern.compile("\\b((public|protected|private)|static|final)(?!.*\\b\\1\\b)(\\s+((public|protected|private)|static|final)(?!.*\\b\\3\\b)){0,2}\\b");
```

Comment:
after testing in Android Studio, even `final static private asdasd` matched and I don't think that's supposed to happen